docs/metadata.md

```diff
@@ -46,9 +42,7 @@ The content is expected to have the following structure:
 }
 ```
 
-#### metadata - String
-
-In case `metadata` is a plain string, we assume it to be the dispute's description and set the dispute creator (`subject.id`) as the plaintiff. No agreement link will be available.
```

Comment:
Note that we only require the `metadata` field to be A JSON object. If we can't parse it, we are displaying the `Could not fetch content from IPFS` error.